spec/classes/puppet_agent_windows_install_spec.rb

```diff
@@ -44,10 +44,17 @@
 
         let(:facts) { facts.merge({:is_pe => true}) }
 
-        it {
-          is_expected.to contain_file('C:\tmp\install_puppet.bat').with_content(
-            %r[#{Regexp.escape("msiexec.exe /qn /norestart /i \"#{values[:appdata]}\\Puppetlabs\\packages\\puppet-agent-#{values[:expect_arch]}.msi\"")}])
-        }
+        context 'with out of date aio_agent_version' do
```

Comment:
This looks like it could be rolled into the other context below (line 68).

Also, we could keep a context where aio_agent_version isn't set to ensure upgrades don't happen.